pkg/server/singleprocess/auth.go

```diff
@@ -123,7 +123,7 @@ func (s *service) tokenFromContext(ctx context.Context) *pb.Token {
 
 // cookieFromRequest returns the server cookie value provided during the request,
 // or blank if none (or a blank cookie) is provided.
-func (s *service) cookieFromRequest(ctx context.Context) string {
+func CookieFromRequest(ctx context.Context) string {
```

Comment:
What's the reason for exporting this instead of keeping it a func pointer? 🤔

Reply:
Other potential waypoint server implementations might not want to reinvent the process of cookie extraction! This function was functionally static anyway (it didn't rely on that pointer receiver at all), so removing that and exporting it makes it easier to consume.